### ISSUE TITLE
Better control over increase/decrease pane count

### ISSUE THREAD

**Issue by ats** [ianyh/Amethyst#248]
I frequently use the increase pane count command to temporarily pull something into my primary pane. I'm looking for more control over _which_ window moves into the main pane --- I think the logic is that the "top" window in the queue (e.g. the one clockwise of the pane) is the one that pops over; I'd love to pop the currently-focused window into the pane, instead. As-is, I spend a lot of time figuring out how to properly position the windows either before or after increasing the pane count.

[Trello Card](https://trello.com/c/K8hKrWXF/86-amethyst-better-control-over-increase-decrease-pane-count)

**Comment by afeblot** [comment]
I agree with the @ats : The common use case is: I have 1 window in the main pane, and I want to bring another specific one there. Ideally, it should indeed bring the focused one when typing OP-, .

If trying to fix the result with OP-Return, the behavior is useless too: Let's say I have Windows 1 in the main pane. OP-, added window 5 whereas I wanted window 6. 
With focus is on Window 6, OP-Return currently swaps 1 and 6, and then 6 and 1, never touching 5. It should either:
* Never remove 1 and swap 5 and 6.
* Consider the main pane as a FIFO: 6 replaces 1 as 1 is the oldest in the main pane. Then with focus on 1, OP-Return: 1 replaces 6 replaces 5 as 5 is now the oldest. And I have the desired result: 1 and 6 in the main pane.
Or is there currently a simpler way? I haven't found any.
